docs/development.rst

```diff
@@ -448,6 +448,145 @@ finally force-push to the remote topic branch on your fork::
+    # the mutation or recombination rates come from other sources. In that case create
+    # additional citations with the appropriate reasons specified (see API documentation
+    # for stdpopsim.citations)
+
+    _assembly_citation = stdpopsim.Citation(
+        doi="FILL ME",
+        year="FILL ME",
+        author="Author et al.",
+        reasons={stdpopsim.CiteReason.ASSEMBLY})
+
+    # Create a genome object
+
+    _genome = stdpopsim.Genome(
+        chromosomes=_chromosomes,
+        assembly_citations=[_assembly_citation])
+
+Once you have a genome object you can create a new `Species` object which contains
+species identifiers, the genome, and default generation time and population size settings
+along with the relevant citation(s). Below is an example species definition for
+Arabidopsis thaliana and a final line of code that registers the species in the catalog.
+
+.. code-block:: python
+
+    _gen_time_citation = stdpopsim.Citation(
+        doi="https://doi.org/10.1890/0012-9658(2002)083[1006:GTINSO]2.0.CO;2",
+        year="2002",
+        author="Donohue",
+        reasons={stdpopsim.CiteReason.GEN_TIME})
+
+    _pop_size_citation = stdpopsim.Citation(
+            doi="https://doi.org/10.1016/j.cell.2016.05.063",
+            year="2016",
+            author="1001GenomesConsortium",
+            reasons={stdpopsim.CiteReason.POP_SIZE})
+
+    _species = stdpopsim.Species(
+        id="AraTha",
+        name="Arabidopsis thaliana",
+        common_name="A. thaliana",
+        genome=_genome,
+        generation_time=1.0,
+        generation_time_citations=[_gen_time_citation],
+        population_size=10**4,
+        population_size_citations=[_pop_size_citation]
+        )
+
+    stdpopsim.register_species(_species)
+
+Once all of this is done, go to the `catalog/__init__.py` file and add a line like the
+one below using the six-letter species identifier. Make sure to keep the comment to
+prevent linting issues.
+
+.. code-block:: python
+
+    from .catalog import PonAbe  #NOQA
+
+--------------------
+Adding a genetic map
+--------------------
+Some species have sub-chromosomal recombination maps available. If there is one for your
+species that you wish to include it is specified in a space delimited file with four
+columns: Chromosome, Position(bp), Rate(cM/Mb), and Map(cM). Each chromosome should be
+placed in a seperate file and with the chromosome id in the file name in such a way that
+it can be programatically parsed out. IMPORTANT: chromosome ids must match those provided
+in the genome definition exactly! Below is an example start to a recombination map file::
+
+    Chromosome Position(bp) Rate(cM/Mb) Map(cM)
+    chr1 32807 5.016134 0
+    chr1 488426 4.579949 0
+
+Once you have the recombination map files formatted, tar and gzip them into a single
+compressed archive. This file will be sent to one of the `stdpopsim` uploaders for
```

Comment:
Might be good to say how the uploader would be contacted and how to coordinate that? Perhaps, opening an issue would work?